### ISSUE TITLE
0.8.0 crash handling magnet link

### ISSUE THREAD

**Issue by factormystic** [picotorrent/picotorrent#221]
![_untitled](https://cloud.githubusercontent.com/assets/43428/13908426/a882cff6-eed8-11e5-8acc-e0526b887ed6.png)

Unfortunately there's a Windows error dialog that takes exclusive focus, so I can't copy/paste the text or click the "create issue" button. The link is
```
magnet:?xt=urn:btih:8343a8a01ff6ff72fb43b344ed3f40838bbfb7f6&dn=Mind.Games.S01E05.The.Cauliflower.Man.720p.WEB-DL.DD5.1.H.264-NT&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80&tr=udp%3A%2F%2Fopen.demonii.com%3A1337&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Fexodus.desync.com%3A6969
```

**Comment by vktr** [comment]
Does this error occur when you click the magnet link from a browser? If so, @zp fixed that in #219 and you could try the binaries [from this build](https://ci.appveyor.com/project/picotorrent/picotorrent/build/238) (click the architecture you're interested in, then click `Artifacts`)

You may have to uninstall your current version before installing the fixed version since they have the same version number.

We will release 0.8.1 soon if you want to wait instead.

**Comment by factormystic** [comment]
Yes, this was clicking the browser link.

**Comment by vktr** [comment]
Should've been fixed in #219

**Comment by factormystic** [comment]
Cool
